src/sagemaker/model_monitor/model_monitoring.py

```diff
@@ -961,6 +963,23 @@ def _wait_for_schedule_changes_to_apply(self):
             if schedule_desc["MonitoringScheduleStatus"] != "Pending":
                 break
 
+    def validate_network_config(self, network_config_dict):
```

Comment:
let's make this a private method so that it can be easily removed in the future

Reply:
Done